mage_ai/streaming/constants.py

```diff
@@ -17,3 +17,4 @@ class SinkType(str, Enum):
     KAFKA = 'kafka'
     KINESIS = 'kinesis'
     OPENSEARCH = 'opensearch'
+    MONGODB = 'mongodb'
```

Comment:
nit: Could you sort it in alphabetical order? (put MONGODB above OPENSEARCH)

Reply:
Done